### PR TITLE
Hackathon landing page

### PR DESCRIPTION
**Pull request by clemlak**
Here is a landing page for the upcoming hackathon.
Some small tweaks could be nice (font sizes and weights) and the content is still using placeholders.

### CODE REVIEW

File: src/components/hackathon/index.tsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    question: "How can I participate in the Huff Hackathon?",
+    answer: "To participate in the Huff Hackathon, you must register on our official website. Registration is free and open to developers of all levels and backgrounds.",
+  },
+  {
+    question: "What is the Huff Hackathon?",
+    answer: "The Huff Hackathon is a virtual event where developers can learn about Huff and EVM programming, build projects, and compete for prizes. It's a great opportunity to learn, network, and collaborate with other developers.",
+  },
+  {
+    question: "When and where is the Huff Hackathon?",
+    answer: "The Huff Hackathon is a virtual event that will take place on the Huff Discord server. The event will begin on Friday, October 1st, 2021, and end on Sunday, October 3rd, 2021.",
```
Comment (devtooligan) [inline src/components/hackathon/index.tsx:64]: oops. wrong date

File: src/components/hackathon/index.tsx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        <h2>Schedule</h2>
+        <Schedule>
+          <tbody>
+            <tr>
+              <td>Event starts</td>
+              <td>Sep 28, 4:00 PM (UTC)</td>
+            </tr>
+            <tr>
+              <td>CTFs &amp; talks</td>
+              <td>Sep 29, TDB (UTC)</td>
```
Comment (devtooligan) [inline src/components/hackathon/index.tsx:109]: i never heard TDB ive only heard TBD and TBA for "to be determined" or "to be announced"

**Comment by devtooligan** [comment]
actually looks like this doesnt include my pr https://github.com/clemlak/huffathon-landing/pull/1/files

**Comment by clemlak** [comment]
> actually looks like this doesnt include my pr https://github.com/clemlak/huffathon-landing/pull/1/files

Yeah I need to include those changes, I had to rewrite the previous landing page from scratch actually so the content was missing.

**Comment by devtooligan** [comment]
cool, if you dont have a chance, i can do it this wknd ill be working on huffathon stuff

**Review by devtooligan — APPROVED** [review]
lgtm

**Pull request merged**